test/unit/useClientRequest.test.js

```diff
@@ -1,5 +1,97 @@
-import { useClientRequest } from '../../src';
+import React from 'react';
+import { act, testHook } from 'react-testing-library';
+import { useClientRequest, ClientContext } from '../../src';
+import { pathToFileURL } from 'url';
```

Comment:
Is this meant to be here? Can't see it used.

Reply:
Nope, my only explanation is VSCode must have added it without me knowing 😬. Thanks!